geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionConfig.java

```diff
@@ -231,6 +231,11 @@
    */
   String SSL_PREFIX = "ssl-";
 
+  /**
+   * The prefix used for Gemfire properties set through java system properties
+   */
+  public static final String GEMFIRE_PREFIX = GeodeGlossary.GEMFIRE_PREFIX;
```

Comment:
I like the idea of keeping the old one but pointing it to the new one.  I would go a step further and add `@deprecated`